### ISSUE TITLE
Support autoIncludes

### ISSUE THREAD

**Issue by rgm3** [freemarker/fmpp#22]
I would like to be able to define the following in config.fmpp:

```
freemarkerAutoIncludes: [
   /path/to/global.ftl,
   /path/to/footer-tools.ftl
]
```

I would expect this to result in a call to [Configuration.setAutoIncludes()](https://freemarker.apache.org/docs/api/freemarker/core/Configurable.html#setAutoIncludes-java.util.List-).

**Comment by ddekany** [comment]
Yeah, FMPP is missing lot of "new" FreeMarker settings... for this one tough, the "borders" setting is a possible workaround (see on http://fmpp.sourceforge.net/settings.html).
